### ISSUE TITLE
Custom picture is not applied when set during onboarding

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#5701]
### Description

1. open status
2. select new account creation
3. set custom picture
4. finish account creation

As result, the picture is not being shown 

<img width="1512" alt="Screenshot 2022-05-11 at 10 27 50" src="https://user-images.githubusercontent.com/82375995/167797034-5201fec5-ed82-43d1-bd18-947f9956bfe1.png">

**Version:** master build 11-May-2022, commit 630c2359d2baf8d661771257903f03ecaf77371b

**Comment by stefandunca** [comment]
Assigning to myself to have a quick look. Profile picture PR depends on it: https://github.com/status-im/status-desktop/pull/5631

**Comment by stefandunca** [comment]
@anastasiyaig just had a look and I could reproduce it and get some data errors. Hence I deleted the old user data folder and it works like a charm now. I can't reproduce anymore. Please check after removing the previous user data completely it seems others had similar migration issues. Let me know if still doesn't work.

**Comment by anastasiyaig** [comment]
@stefandunca yes, it seems that migrations do not work here. We need to think of that, if we are planning a release - do we want to make users delete their folder and re-import keys? Or allow to upgrade ?
